requirements.txt

```diff
@@ -0,0 +1,6 @@
+click==7.0
+tensorflow-datasets==1.2.0
+polyaxon-client==0.5.6
+larq==0.7.4
+zookeeper==0.5.5
+tensorflow-addons==0.6.0
```

Comment:
If we remove the autoaugment code, we can remove this too. Also this would break with TF 1
```suggestion
```